src/zeit/retresco/connection.py

```diff
@@ -119,8 +119,26 @@ def _get_intextlink_data(self, content, timeout):
         except (KeyError, requests.Timeout):
             return {}
 
-    def get_article_keywords(self, content, timeout=None):
-        response = self._get_intextlink_data(content, timeout)
+    def _get_intextlink_data_preview(self, content, timeout):
+        # In contrast to _get_intextlink_data, the tms computes the
+        # the intext_links from the given content, and does not look it up
+        tms_content = zeit.retresco.interfaces.ITMSRepresentation(content)()
+        try:
+            return self._request(
+                'POST /in-text-linked-documents-preview',
+                json=tms_content,
+                timeout=timeout)
+        except requests.Timeout:
+            log.info(
+                '/in-text-linked-documents-preview request for %s timed out',
+                content.uniqueId)
+            return {}
+
+    def get_article_keywords(self, content, timeout=None, post_data=False):
```

Comment:
Können wir das etwas weniger mechanisch und mehr inhaltlich benennen? Sowas in die Richtung `published=True`? (Oh und im `ITMS` sollte man den Parameter noch nachtragen und dokumentieren.)

Reply:
Jo gerne, ich habe mir da bei der Benamsung einen abgebrochen :)